cmd/block-generator/run_generator.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Demonstrate how to run the generator and connect it to indexer.
+
+POSTGRES_CONTAINER=generator-test-container
+POSTGRES_PORT=15432
+POSTGRES_DATABASE=generator_db
+
+function start_postgres() {
+   # Start postgres container...
+   docker run \
+     -d \
+     --name $POSTGRES_CONTAINER \
+     -e POSTGRES_USER=algorand \
+     -e POSTGRES_PASSWORD=algorand \
+     -e PGPASSWORD=algorand \
+     -p $POSTGRES_PORT:5432 \
```

Comment:
could you make a constant for 5432? what is it for btw?

Reply:
This script is just an MVP so I don't think we need to worry about cleaning it up, a more advanced test driver is going to be needed: https://github.com/algorand/indexer/issues/443